sunpy/net/scraper.py

```diff
@@ -9,7 +9,7 @@
 from urllib.error import HTTPError
 from urllib.parse import urlsplit
 from urllib.request import urlopen
-
+from urllib.error import URLError
```

Comment:
Can you make sure the style checks are passing?

Reply:
Sure but what does that have to do with the style?

Reply:
There should be a section about this in our documentation. You don't need to understand them but you do need to fix them.